scripts/time-drift.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
+for i in $(serviced host list| awk '{print $4}' | grep -v Name | grep -v $HOSTNAME); do echo ntpdate \-q $i; ntpdate -q $i; done
```

Comment:
I agree, thank your for catching this and I have updated the code for 10 seconds.